website/source/docs/guides/acl.html.md

```diff
@@ -1,288 +1,142 @@
 ---
 layout: "docs"
-page_title: "ACL System"
+page_title: "Bootstrapping ACLs"
```

Comment:
Having the top level ACL "guide" come here with the other guides nested underneath seems wrong for some reason.

Maybe the top level ACL guide page should be more of a landing page with some general information about the guides. Then this would be just one of them underneath.

The landing page guide could also provide some ordering to point out that first you should do a basic setup, then have additional guides for multi-dc etc.

Reply:
As discussed during our call, we could update the "ACL" guide to be a landing page like https://www.consul.io/docs/guides/index.html. The new landing page could have the links to the 3 guides and reference the two documents. 

Generally, I think its a great idea to give people more direction. Even though the guides are moving, I think the effort/impact is still worth the changes. 

@mkeeler For now I am going to close this comment and create a new PR for the reorg and new outline. Maybe we can work together on this?